### ISSUE TITLE
Add "Why am I getting attribute warnings?" to the FAQ

### ISSUE THREAD

**Issue by mxstbr** [styled-components/styled-components-website#150]
Basically, my answer in this issue https://github.com/styled-components/styled-components/issues/1198#issuecomment-336628848 needs to be cleaned up and put into our FAQ!

**Comment by morajabi** [comment]
Should we just explain it over react-router just like you did in that issue or make it more generalized?

**Comment by mxstbr** [comment]
More generalised I think?

**Comment by morajabi** [comment]
Yea and then we can take react router as an example since it's the most common cause of this confusion.

**Comment by rodocite** [comment]
I submitted a PR for adding this documentation. I peeped the RR source for Link (https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/modules/Link.js#L75) and it is as @mxstbr described. I kept his explanation but adjusted the examples slightly to provide a bit more detail.

I think there are many instances where this can happen aside from RR. People will probably run into this as they're building integration tests so I took a more generalized explanation but kept the "Link" example.
